client/layout/guided-tours/index.js

```diff
@@ -15,14 +15,20 @@ import QueryPreferences from 'components/data/query-preferences';
 import RootChild from 'components/root-child';
 import { getGuidedTourState } from 'state/ui/guided-tours/selectors';
 import { getLastAction } from 'state/ui/action-log/selectors';
-import { getSectionName, isSectionLoading } from 'state/ui/selectors';
-import { nextGuidedTourStep, quitGuidedTour } from 'state/ui/guided-tours/actions';
+import { getInitialQueryArguments, getSectionName, isSectionLoading } from 'state/ui/selectors';
+import { nextGuidedTourStep, quitGuidedTour, resetGuidedTourHistory } from 'state/ui/guided-tours/actions';
 
 class GuidedTours extends Component {
 	shouldComponentUpdate( nextProps ) {
 		return this.props.tourState !== nextProps.tourState;
 	}
 
+	componentWillReceiveProps( nextProps ) {
+		if ( 'reset' === nextProps.queryArgs.tour && this.props.queryArgs.tour !== 'reset' ) {
```

Comment:
Minor, but we don't require yoda-style conditions in Calypso. I'm fine with it, but let's make sure we use the same style for both checks on the line. :)

Reply:
😁